lib/rack/response.rb

```diff
@@ -56,6 +61,12 @@ def initialize(body = nil, status = 200, header = {})
       yield self if block_given?
     end
 
+    attr_accessor :length, :status, :body
+    attr_reader :headers
+
+    # Deprecated.
```

Comment:
If we are using yard, should this be `@deprecated`?

Reply:
Yes.